draft-bbs-signatures.md

```diff
@@ -1121,13 +1148,13 @@ The following ciphersuite is based on the BLS12-381 elliptic curve defined in Se
 
 - hash: SHAKE-256 as defined in [@!SHA3].
 
-- xof\_no\_of\_bytes: 64.
+- expand\_length: 48.
```

Comment:
I don't think this is used anymore

Reply:
I'm really sorry about that! Fixed. Thanks!